README.md

```diff
@@ -1,2 +1,51 @@
-# pairing-data-engineer
-Pairing exercise for a data engineer
+Data Engineer Pairing Exercise
+==============================
+
+
+# The Task
+
+The task involves developing a data pipeline and its underlying infrastructure,  in order to complete the following user story, using sample data sources that will be provided.
```

Comment:
Change text to: "The task involves developing a data pipeline and its underlying infrastructure. In order to complete the following user story sample data sources will be provided."

Reply:
done